packages/common/src/types/dashboard.ts

```diff
@@ -52,6 +53,21 @@ export type DashboardChartTileProperties = {
     };
 };
 
+export type DashboardSqlChartTileProperties = {
+    type: DashboardTileTypes.SQL_CHART;
+    properties: {
+        title?: string;
+        savedSqlUuid: string | null;
+        chartName?: string | null;
```

Comment:
I made this required, that's all!
```suggestion
        chartName: string;
```